[08-May-24 18:07:00 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:07:00 - INFO] - bot - Bot stopped.
[08-May-24 18:07:09 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:07:09 - INFO] - bot - Lets roll
[08-May-24 18:07:12 - ERROR] - pyrogram.dispatcher - Not a gzipped file (b'\xf7\xd7')
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 40, in start_command
    decoded_message = await decode(comm_clean)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 188, in decode
    decompressed_string = gzip.decompress(compressed_bytes).decode("utf-8")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 601, in decompress
    if _read_gzip_header(fp) is None:
       ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 428, in _read_gzip_header
    raise BadGzipFile('Not a gzipped file (%r)' % magic)
gzip.BadGzipFile: Not a gzipped file (b'\xf7\xd7')
[08-May-24 18:09:00 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:09:00 - INFO] - bot - Bot stopped.
[08-May-24 18:09:08 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:09:08 - INFO] - bot - Lets roll
[08-May-24 18:09:32 - ERROR] - pyrogram.dispatcher - Not a gzipped file (b'\xe7\xde')
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 40, in start_command
    decoded_message = await decode(comm_clean)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 188, in decode
    decompressed_string = gzip.decompress(compressed_bytes).decode("utf-8")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 601, in decompress
    if _read_gzip_header(fp) is None:
       ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 428, in _read_gzip_header
    raise BadGzipFile('Not a gzipped file (%r)' % magic)
gzip.BadGzipFile: Not a gzipped file (b'\xe7\xde')
[08-May-24 18:10:17 - ERROR] - pyrogram.dispatcher - 'NoneType' object has no attribute 'strip'
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\callback_query_handler.py", line 21, in callback_handler
    decoded_message = await decode(data)
                      ^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 183, in decode
    base64_string = base64_string.strip("=")
                    ^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'strip'
[08-May-24 18:11:56 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:11:56 - INFO] - bot - Bot stopped.
[08-May-24 18:12:04 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:12:04 - INFO] - bot - Lets roll
[08-May-24 18:12:35 - ERROR] - pyrogram.dispatcher - Not a gzipped file (b'\xf7\xd7')
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 40, in start_command
    decoded_message = await decode(comm_clean)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 188, in decode
    decompressed_string = gzip.decompress(compressed_bytes).decode("utf-8")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 601, in decompress
    if _read_gzip_header(fp) is None:
       ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 428, in _read_gzip_header
    raise BadGzipFile('Not a gzipped file (%r)' % magic)
gzip.BadGzipFile: Not a gzipped file (b'\xf7\xd7')
[08-May-24 18:14:31 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:14:31 - INFO] - bot - Bot stopped.
[08-May-24 18:14:41 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:14:41 - INFO] - bot - Lets roll
[08-May-24 18:18:13 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:18:13 - INFO] - bot - Bot stopped.
[08-May-24 18:18:21 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:18:21 - INFO] - bot - Lets roll
[08-May-24 18:18:58 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:18:58 - INFO] - bot - Bot stopped.
[08-May-24 18:19:07 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:19:07 - INFO] - bot - Lets roll
[08-May-24 18:20:05 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:20:05 - INFO] - bot - Bot stopped.
[08-May-24 18:20:13 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:20:13 - INFO] - bot - Lets roll
[08-May-24 18:20:53 - ERROR] - pyrogram.dispatcher - Not a gzipped file (b'\xf7\xd7')
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 40, in start_command
    decoded_message = await decode(comm_clean)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 188, in decode
    decompressed_string = gzip.decompress(compressed_bytes).decode("utf-8")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 601, in decompress
    if _read_gzip_header(fp) is None:
       ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 428, in _read_gzip_header
    raise BadGzipFile('Not a gzipped file (%r)' % magic)
gzip.BadGzipFile: Not a gzipped file (b'\xf7\xd7')
[08-May-24 18:21:43 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:21:43 - INFO] - bot - Bot stopped.
[08-May-24 18:21:51 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:21:51 - INFO] - bot - Lets roll
[08-May-24 18:22:14 - ERROR] - pyrogram.dispatcher - Not a gzipped file (b'\xf7\xd7')
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 40, in start_command
    decoded_message = await decode(comm_clean)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 188, in decode
    decompressed_string = gzip.decompress(compressed_bytes).decode("utf-8")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 601, in decompress
    if _read_gzip_header(fp) is None:
       ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 428, in _read_gzip_header
    raise BadGzipFile('Not a gzipped file (%r)' % magic)
gzip.BadGzipFile: Not a gzipped file (b'\xf7\xd7')
[08-May-24 18:22:37 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:22:37 - INFO] - bot - Bot stopped.
[08-May-24 18:22:46 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:22:46 - INFO] - bot - Lets roll
[08-May-24 18:22:48 - ERROR] - pyrogram.dispatcher - Not a gzipped file (b'\xf7\xd7')
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 40, in start_command
    decoded_message = await decode(comm_clean)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 188, in decode
    decompressed_string = gzip.decompress(compressed_bytes).decode("utf-8")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 601, in decompress
    if _read_gzip_header(fp) is None:
       ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 428, in _read_gzip_header
    raise BadGzipFile('Not a gzipped file (%r)' % magic)
gzip.BadGzipFile: Not a gzipped file (b'\xf7\xd7')
[08-May-24 18:26:08 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:26:08 - INFO] - bot - Bot stopped.
[08-May-24 18:26:17 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:26:17 - INFO] - bot - Lets roll
[08-May-24 18:26:24 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:26:24 - INFO] - bot - Bot stopped.
[08-May-24 18:26:33 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:26:33 - INFO] - bot - Lets roll
[08-May-24 18:27:08 - ERROR] - pyrogram.dispatcher - cannot access local variable 'command_clean' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 38, in start_command
    comm_clean = command_clean(text)
                 ^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'command_clean' where it is not associated with a value
[08-May-24 18:28:16 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:28:16 - INFO] - bot - Bot stopped.
[08-May-24 18:28:25 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:28:25 - INFO] - bot - Lets roll
[08-May-24 18:31:15 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:31:15 - INFO] - bot - Bot stopped.
[08-May-24 18:31:24 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:31:24 - INFO] - bot - Lets roll
[08-May-24 18:32:26 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:32:26 - INFO] - bot - Bot stopped.
[08-May-24 18:32:35 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:32:35 - INFO] - bot - Lets roll
[08-May-24 18:33:50 - ERROR] - pyrogram.dispatcher - Not a gzipped file (b'\x99\xa9')
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 55, in start_command
    if not ids:
          ^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 105, in extract_ids
    string = await decode(base64_string)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\helper_func.py", line 188, in decode
    decompressed_string = gzip.decompress(compressed_bytes).decode("utf-8")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 601, in decompress
    if _read_gzip_header(fp) is None:
       ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manfr\anaconda3\Lib\gzip.py", line 428, in _read_gzip_header
    raise BadGzipFile('Not a gzipped file (%r)' % magic)
gzip.BadGzipFile: Not a gzipped file (b'\x99\xa9')
[08-May-24 18:38:33 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:38:33 - INFO] - bot - Bot stopped.
[08-May-24 18:38:41 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:38:41 - INFO] - bot - Lets roll
[08-May-24 18:39:30 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:39:30 - INFO] - bot - Bot stopped.
[08-May-24 18:39:40 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:39:40 - INFO] - bot - Lets roll
[08-May-24 18:40:54 - ERROR] - pyrogram.dispatcher - object of type 'NoneType' has no len()
Traceback (most recent call last):
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\manfr\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "C:\WorkSpace\Apps\Bots\File-Sharing-Bot\plugins\start_command.py", line 54, in start_command
    if len(decoded_message) > 7 and not starts_with_bot_username(client.me.username, decoded_message):
       ^^^^^^^^^^^^^^^^^^^^
TypeError: object of type 'NoneType' has no len()
[08-May-24 18:41:43 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-May-24 18:41:43 - INFO] - bot - Bot stopped.
[08-May-24 18:41:51 - INFO] - bot - Bot Running..!

Created by 
https://t.me/stilloid
[08-May-24 18:41:51 - INFO] - bot - Lets roll
